### DISCUSSION TITLE
Welcome to SpectralSuite Discussions!

### DISCUSSION THREAD

**Discussion by andrewreeman** [andrewreeman/SpectralSuite#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hsoiland** [comment]
Hi all, Im zonked. Im an hyperglitch producer. I'm hoping to extend the feature set of spectral suite, adding bits and pieces that would allow me to push my music production and sound design in new directions or work on concepts I have. And work on the UI/interop a little to make it easier to do automation. I predominantly work in the medical space as an SDE3.

**Reply by andrewreeman** [reply]
Welcome 🤗,

Improving the automation on here would be a great addition. I've not had the chance to focus on it much during testing etc so I'm sure there are areas where it totally breaks! 

Any improvement on UI is welcome too as it's obvious just the defaults 😅. Not much thought on UX, accessibility or making it remotely pretty. 

Thanks for showing interest in this!
